rakelib/plugins-metadata.json

```diff
@@ -121,7 +121,8 @@
   },
   "logstash-filter-elastic_integration": {
     "default-plugins": false,
-    "skip-list": false
+    "skip-list": false,
+    "oss-excluded": true
```

Comment:
wooh, difficult namings :( I would prefer `skipped`, `default-plugin` but we already made them.

Reply:
skip-oss instead of "oss-excluded"